### ISSUE TITLE
After table drop tree is closed

### ISSUE THREAD

**Issue by ManPython** [HeidiSQL/HeidiSQL#1804]
**Preconditions**

* HeidiSQL version: Revision 6676 (yours: 6676)
Compiled: 2023-05-08 (yours: 2023-05-08)
 
* Database type and version: MariaDB/MySQL - other, eg. SQLite
* OS: W11

**Describe the bug**
HeidiSQL not listing Rows, size for tables, just only after when tables was selected to show data.

**To Reproduce**

1. Click RMB on table, then drop
- in my experience behavior is that tree is closed and user must open again to operate on DB or other tables, eg. drop other, etc,
-- it's problem coz if long list of tables, then must find right.. 

**Screenshots or Crash reports**

**Comment by ansgarbecker** [comment]
Duplicate of #16
